README.md

```diff
@@ -13,23 +13,24 @@ REST API for managing cache control information for pages within the legacy CMS
-| HEALTHCHECK_INTERVAL               | 30s                                                        | Time between self-healthchecks (`time.Duration` format)                                                             |
-| HEALTHCHECK_CRITICAL_TIMEOUT       | 90s                                                        | Time to wait until an unhealthy dependent propagates its
+| Environment variable         | Default                         | Description                                                       |
+|------------------------------|---------------------------------|-------------------------------------------------------------------|                
+| ZEBEDEE_URL                  | localhost:8082                  | Zebedde host address and port for authentication                  |
+| BIND_ADDR                    | :29100                          | The host and port to bind to                                      |
+| MONGODB_BIND_ADDR            | localhost:27017                 | The MongoDB bind address                                          |
+| MONGODB_USERNAME             |                                 | The MongoDB Username                                              |
+| MONGODB_PASSWORD             |                                 | The MongoDB Password                                              |
+| MONGODB_DATABASE             | cache                           | The MongoDB database                                              |
+| MONGODB_COLLECTIONS          | CacheTimesCollection:cachetimes | The MongoDB collections                                           |                           
+| MONGODB_REPLICA_SET          |                                 | The name of the MongoDB replica set                               |
+| MONGODB_ENABLE_READ_CONCERN  | false                           | Switch to use (or not) majority read concern                      |
+| MONGODB_ENABLE_WRITE_CONCERN | true                            | Switch to use (or not) majority write concern                     |
+| MONGODB_CONNECT_TIMEOUT      | 5s                              | The timeout when connecting to MongoDB (`time.Duration` format)   |
+| MONGODB_QUERY_TIMEOUT        | 15s                             | The timeout for querying MongoDB (`time.Duration` format)         |
+| MONGODB_IS_SSL               | false                           | Switch to use (or not) TLS when connecting to mongodb             |
+| GRACEFUL_SHUTDOWN_TIMEOUT    | 5s                              | The graceful shutdown timeout in seconds (`time.Duration` format) |
+| HEALTHCHECK_INTERVAL         | 30s                             | Time between self-healthchecks (`time.Duration` format)           |
+| HEALTHCHECK_CRITICAL_TIMEOUT | 90s                             | Time to wait until an unhealthy dependent propagates its          
```

Comment:
This isn't your fault, but it looks like at some point this description was cropped. Could you fix it, please? It should say:
```
Time to wait until an unhealthy dependent propagates its state to make this app unhealthy (`time.Duration` format)
```